recipes/e-antic/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+export CFLAGS="-O2 -g $CFLAGS"
+export CXXFLAGS="-O2 -g $CXXFLAGS"
+export LD_LIBRARY_PATH="${PREFIX}/lib:${LD_LIBRARY_PATH}"
+export DYLD_FALLBACK_LIBRARY_PATH="${PREFIX}/lib:${DYLD_FALLBACK_LIBRARY_PATH}"
```

Comment:
All four export lines should not be required. Is something failing without them?

Reply:
Thanks for the feedback :) I had to put these in other recipes in the past. Let me try what happens if I remove them.

Reply:
@xhochy: Guess you were right. Looks good to me without these lines.

Reply:
With the recent migration to the new compilers most of these flags are now already set automatically. Also, the `LD_LIBRARY_PATH` related things should be automatically correct by `conda` itself.